src/chord.cpp

```diff
@@ -188,13 +188,32 @@ int Chord::PositionInChord(Note *note)
 
 void Chord::GetYExtremes(int &yMax, int &yMin)
 {
+    bool childNoteFound = false;
+    int y1;
     ListOfObjects *childList = this->GetList(this); // make sure it's initialized
     assert(childList->size() > 0);
 
-    // The first note is the bottom
-    yMin = childList->front()->GetDrawingY();
-    // The last note is the top
-    yMax = childList->back()->GetDrawingY();
+    // Iterate through list of child Note objects to find lowest- and highest-pitched; not dependably in order
+    for (ListOfObjects::iterator it = childList->begin(); it != childList->end(); it++) {
```

Comment:
If you have a [C4, E4, G4] triad and drag the middle E down an octave in an editing interface so it's now [C4, E3, G4], is there a way to trigger FilterList from the functor calls in Page::LayOutPitchPos()?